diagnostic.md

```diff
@@ -40,15 +50,17 @@ methods.
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+document object model
+tree view set up where each piece is a part of the document, describes structure of object
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface to render web pages for you to view and interact with.